backend/.env.dist

```diff
@@ -1,4 +1,4 @@
-CONFIG_VERSION=v11.2022-10-27
+CONFIG_VERSION=v12.2022-11-10
```

Comment:
do we miss an adjustment in `deployment/bare_metal/.env.dist` ?

Reply:
done